.travis.yml

```diff
@@ -1,12 +1,10 @@
 language: go
 sudo: true
 go:
-  - 1.11.x
-  - 1.12.x
```

Comment:
I think this is fine, but we should notify the presentation folks that we made this change

Reply:
they're on 1.13 and will go to 1.14 soon (there is some version lag for their repo). So, this should be okay.  If needed, and they don't move fast enough, we can add 1.13.